mcp/utils.go

```diff
@@ -775,3 +775,8 @@ func ParseStringMap(request CallToolRequest, key string, defaultValue map[string
 	v := ParseArgument(request, key, defaultValue)
 	return cast.ToStringMap(v)
 }
+
+// ToBoolPtr returns a pointer to the given boolean value
+func ToBoolPtr(b bool) *bool {
+	return &b
+}
```

Comment:
ToBoolPtr is a more semantic name. This is fine IMO and is followed in other Go projects for the same reason, to distinguish between not specified and zero values.

https://stackoverflow.com/questions/54823690/why-does-kubernetes-internally-use-string-pointers-rather-than-strings